scripts/docker_push.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
 
 for tag in $*;do
-    echo "Tag ${IMAGE_NAME}:${DOCKER_TMP_TAG} as ${IMAGE_NAME}:${tag}"
-    docker tag "${IMAGE_NAME}:${DOCKER_TMP_TAG}" "${IMAGE_NAME}:${tag}"
-    echo "Push ${IMAGE_NAME}:${tag}"
-    docker push "${IMAGE_NAME}:${tag}"
+    echo "Tag ${IMAGE_NAME_WORKER}:${DOCKER_TMP_TAG} as ${IMAGE_NAME_WORKER}:${tag}"
+    docker tag "${IMAGE_NAME_WORKER}:${DOCKER_TMP_TAG}" "${IMAGE_NAME_WORKER}:${tag}"
+    echo "Push ${IMAGE_NAME_WORKER}:${tag}"
+    docker push "${IMAGE_NAME_WORKER}:${tag}"
+
+    echo "Tag ${IMAGE_NAME_CLI}:${DOCKER_TMP_TAG} as ${IMAGE_NAME_CLI}:${tag}"
+    docker tag "${IMAGE_NAME_CLI}:${DOCKER_TMP_TAG}" "${IMAGE_NAME_CLI}:${tag}"
+    echo "Push ${IMAGE_NAME_CLI}:${tag}"
+    docker push "${IMAGE_NAME_CLI}:${tag}"
```

Comment:
you have the same code twice here, use a loop

Reply:
Done